WTWSMS/events/WtWSMS_schisms.txt

```diff
@@ -258,25 +254,51 @@ character_event = {
 	picture = GFX_evt_icon
 	
 	trigger = {
-		year = 950
+		year = 950 #Why this year? Temporary anyway.
 		religion = nicene 
 		controls_religion = yes
+		has_landed_title = k_nicene
 		NOT = { has_global_flag = great_schism }
 	}
 	
 	option = {
 		name = EVTOPTAHRE.4
 
-		set_parent_religion = { religion = arian parent = catholic }
-		set_parent_religion = { religion = valentinianism parent = catholic }
-		set_parent_religion = { religion = donatism parent = catholic }
-		set_parent_religion = { religion = docetic parent = orthodox }
-		set_parent_religion = { religion = ebionites parent = orthodox }
-		set_parent_religion = { religion = marcionism parent = orthodox }
-		set_parent_religion = { religion = binitarian parent = orthodox }
+		if = {
+			limit = { religion = catholic }
+			set_parent_religion = { religion = arian parent = catholic }
+		}
+		if = {
+			limit = { religion = catholic }
+			set_parent_religion = { religion = valentinianism parent = catholic }
+		}
+		if = {
+			limit = { religion = catholic }
+			set_parent_religion = { religion = donatism parent = catholic }
+		}
+		if = {
+			limit = { religion = orthodox }
+			set_parent_religion = { religion = docetic parent = orthodox }
+		}
+		if = {
+			limit = { religion = orthodox }
+			set_parent_religion = { religion = ebionites parent = orthodox }
+		}
+		if = {
+			limit = { religion = orthodox }
+			set_parent_religion = { religion = marcionism parent = orthodox }
+		}
+		if = {
+			limit = { religion = orthodox }
+			set_parent_religion = { religion = binitarian parent = orthodox }
+		}
```

Comment:
I think the limit = { religion = xxx } will never match the religion of the character (head of nicene religion), and old code should be kept.